modules/identity/src/Volo.Abp.Identity.Application/Volo/Abp/Identity/ProfileAppService.cs

```diff
@@ -20,9 +21,13 @@ public ProfileAppService(IdentityUserManager userManager)
 
         public virtual async Task<ProfileDto> GetAsync()
         {
-            return ObjectMapper.Map<IdentityUser, ProfileDto>(
-                await UserManager.GetByIdAsync(CurrentUser.GetId())
-            );
+            var currentUser = await UserManager.GetByIdAsync(CurrentUser.GetId());
+
+            var profile = ObjectMapper.Map<IdentityUser, ProfileDto>(currentUser);
+            profile.IsExternalLoggedIn = currentUser.IsExternal;
```

Comment:
If you rename `profile.IsExternalLoggedIn` to `currentUser.IsExternal` then you don't need to manually map.